msfs/src/sim_connect.rs

```diff
@@ -534,6 +534,55 @@ impl<'a> SimConnect<'a> {
         }
         Ok(())
     }
+
+    /// Load a .FLT file from disk
+    pub fn load_flight(&mut self, flight_file_path: &str) -> Result<()> {
+        let flight_file_path = std::ffi::CString::new(flight_file_path).unwrap();
+
+        unsafe {
+            map_err(sys::SimConnect_FlightLoad(
+                self.handle,
+                flight_file_path.as_ptr(),
+            ))?;
+        }
+        Ok(())
+    }
+
+    /// Save the current sim state to a .FLT file
+    pub fn save_flight(
+        &mut self,
+        flight_file_path: &str,
+        title: Option<&str>,
+        description: Option<&str>,
+    ) -> Result<()> {
+        let flight_file_path = std::ffi::CString::new(flight_file_path).unwrap();
+        let title = title.map(|x| std::ffi::CString::new(x).unwrap());
+        let description = description.map(|x| std::ffi::CString::new(x).unwrap());
+
+        unsafe {
+            map_err(sys::SimConnect_FlightSave(
+                self.handle,
+                flight_file_path.as_ptr(),
+                title.map(|x| x.as_ptr()).unwrap_or(std::ptr::null()),
+                description.map(|x| x.as_ptr()).unwrap_or(std::ptr::null()),
```

Comment:
This needs to be fixed. Otherwise the string will be deallocated before the function gets actually called.
```suggestion
                title.as_ref().map(|x| x.as_ptr()).unwrap_or(std::ptr::null()),
                description.as_ref().map(|x| x.as_ptr()).unwrap_or(std::ptr::null()),
```